tdrs-frontend/src/components/EditProfile/EditProfile.jsx

```diff
@@ -139,76 +139,76 @@ function EditProfile() {
     return <Redirect to="/request" />
   }
 
-    return (
-        <>
-          <p className="margin-top-1 margin-bottom-4">
-            Please enter your information to request access from an OFA
-            administrator
-          </p>
-          <form className="usa-form" onSubmit={handleSubmit}>
-            <div
-              className={`usa-error-message ${
+  return (
```

Comment:
This entire file is just a linter fix. Moving everything to 2 spaces.

Reply:
Right. Not a blocker, but curious -- is there a reason the linters/CI wasn't failing before if the spacing was off?

Reply:
This is a great question. I'm not sure why my linter is making these changes. I think it might have something to do with my Prettier set up that automatically fixes any linter errors. I am going to try and take a look at this next week so both Aaron and I can have the same rules set up and we don't get these types of changes.